chat/basics/android.md

```diff
@@ -485,6 +532,10 @@ chatContainer.editMessage(message, "New Body", new SaveCallback<Message>() {
 });
 ```
 
+The `editMessage` function also save the message to local cache store.
+`editMessage` is similar to `addMessage` in that there are different
+callbacks indicating whether the result is a cached one.
```

Comment:
It mention `addMessage` and `editMessage` are similar, so what if developer want to retry the edit message? call `editMessage` again? And give up call `deleteMessage`? a bit confused as I assume calling `deleteMessage` would not just delete the cached edits, but the message on server?